apps/sensenet/src/components/content-list/actions-field.tsx

```diff
@@ -9,7 +9,7 @@ export const ActionsField: React.FC<{
 }> = ({ onOpen }) => {
   const globalClasses = useGlobalStyles()
   return (
-    <TableCell component="div" className={clsx(globalClasses.centered, globalClasses.virtualizedCellStyle)}>
+    <TableCell className={clsx(globalClasses.centered, globalClasses.virtualizedCellStyle)}>
```

Comment:
there are warnings in the unit tests (and I think in the browser console too) because of these TableCell changes:
```Warning: validateDOMNesting(...): <td> cannot appear as a child of <div>.```